delta/plugins/elasticsearch/src/main/resources/elasticsearch.conf

```diff
@@ -45,4 +45,7 @@ plugins.elasticsearch {
   }
   # the query config used when fetching all scoped events to obtain metrics
   metrics-query = ${app.defaults.query}
+  # when performing aggregations for default views, defines the maximum number
+  # of buckets for the aggregation
+  listing-bucket-size = 500
```

Comment:
By default on aggregations, ES returns the top 10 of values.
This allows to configure it.